Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/FunctionsController.cs

```diff
@@ -184,22 +184,21 @@ public void TeardownFunctions()
 
         private static async Task<string> GetToolPath()
         {
-            string npmPrefix = await GetNpmPrefix().ConfigureAwait(false);
-            string toolsFolder = Path.Combine(
-                npmPrefix,
-                @"node_modules\azure-functions-core-tools\bin");
-
-            if (!Directory.Exists(toolsFolder))
-            {
-                throw new FunctionStartupException(
-                    $"Azure Functions runtime not found at {toolsFolder}. Have you run: " +
-                    "'npm install -g azure-functions-core-tools@3 --unsafe-perm true'?");
-            }
```

Comment:
We could keep this logic to throw a `FunctionStartupException` if the runtime is not installed.

I think we could check for a non-zero exit code for both `where` and `which` (which would indicate the runtime is not installed)

Reply:
Reinstated the exception thrown if the runtime is not installed👍🏻